pkg/reconciler/buildrun/resources/taskrun_test.go

```diff
@@ -531,11 +530,11 @@ var _ = Describe("GenerateTaskrun", func() {
 						paramOutputInsecureFound = true
 						Expect(param.Value.StringVal).To(Equal("false"))
 
-					case "BUILDER_IMAGE":
+					case "builder-image":
 						paramBuilderImageFound = true
 						Expect(param.Value.StringVal).To(Equal(builderImage.Image))
 
-					case "DOCKERFILE":
+					case "dockerfile":
 						paramDockerfileFound = true
 						Expect(param.Value.StringVal).To(Equal(dockerfile))
```

Comment:
Given these are now "normal" strategy parameters, I guess they can be removed from this test case.